docker-compose-entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+# first arg is `-f` or `--some-option`
+if [ "${1#-}" != "$1" ]; then
+	set -- docker-compose "$@"
+fi
```

Comment:
You could avoid doing the extra call to if by writing this as a one-liner:

[ "${1#-}" == "$1" ] || set -- docker-compose "$@"

Reply:
IMHO, your suggestion lacks readability, which is important in docker-compose.

And the second point on that, is that actually this code is just a copy from https://github.com/docker-library/docker/blob/69ab1f338fbbdc57fa6dae39308bcd2fc98560a8/18.09/docker-entrypoint.sh#L5